# service message for pd parameter tuning of the joint controller

float64 kp
float64 kv
---
bool setting_is_done

